modules/dev-test-lab/labs/.test/common/main.test.bicep

```diff
@@ -222,7 +222,7 @@ module testDeployment '../../main.bicep' = {
         }
       }
     ]
-    notificationChannels: [
+    notificationchannels: [
```

Comment:
Did you intend to remove the camel casing here?

Reply:
Hey, to align with the provider documentation: https://learn.microsoft.com/en-us/azure/templates/microsoft.devtestlab/labs/notificationchannels?pivots=deployment-language-bicep#resource-format